include/deal.II/differentiation/sd/symengine_optimizer.h

```diff
@@ -1124,6 +1124,9 @@ namespace Differentiation
        * @tparam TensorType The type of tensor to be evaluated and returned
        *         (i.e. Tensor or SymmetricTensor).
        * @param[in] symbol_tensor The symbolic tensor that is to be evaluated.
+       * @param[in] cached_evaluation The vector that stores the numerical
```

Comment:
Here and below, since it's an input vector, maybe saying
```suggestion
       * @param[in] cached_evaluation A vector that stores the numerical
```
would be more appropriate because it lives in user space (right?). But I wouldn't know what I should be putting into this argument. Could you explain?